modules/nw-sriov-supported-devices.adoc

```diff
@@ -77,6 +77,36 @@
 |8086
 |1593
 
+|Intel
+|Ice E810-XXV Backplane
+|8086
+|1599
+
+|Intel
+|Ice E823L Backplane
+|8086
+|124c
+
+|Intel
+|Ice E823L SFP
+|8086
+|124d
+
+|Intel
```

Comment:
I think you have a duplicate entry here. This entry is the same as the one before.

Reply:
Oops! Nice catch. Fixed. Thank you!